projects/plugins/crm/src/automation/commons/triggers/events/class-event-deleted.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Jetpack CRM Automation Event_Deleted trigger.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+/**
+ * Adds the Event_Deleted class.
+ */
+class Event_Deleted extends Base_Trigger {
+
+	/**
+	 * Get the slug name of the trigger.
+	 * @return string
+	 */
+	public static function get_slug(): string {
+		return 'jpcrm/event_deleted';
+	}
+
+	/**
+	 * Get the title of the trigger.
+	 * @return string
+	 */
+	public static function get_title(): ?string {
+		return __( 'Delete Event', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the description of the trigger.
+	 * @return string
+	 */
+	public static function get_description(): ?string {
+		return __( 'Triggered when a event is deleted', 'zero-bs-crm' );
```

Comment:
It's probably not worth to take too seriously until we have the UI in place, but our messages could probably be phrased slightly different to make more sense to users.

E.g.: `Start a workflow when an event is deleted`

```suggestion
		return __( 'Triggered when an event is deleted', 'zero-bs-crm' );
```